README.md

```diff
@@ -1,17 +1,94 @@
 ## Firefox Application Services
 
-### Contents
+_A platform for building cloud-powered applications that target Firefox users_
 
-* [fxa-client](components/fxa-client) - cross compiled FxA Rust client that can work with Firefox Sync keys and more
-* [sandvich-ios](samples/sandvich-ios) - Example apps that use SDKs built on top of `fxa-client` to demonstrate a FxA login flow.
-* [sync15](components/sync15) - Sync 1.5 adapter
-* [libs](libs) - libs directory has build scripts for native libraries
-* [docs](docs) - documentation sources 
-* [website](website) - website built from documentation sources
+### What's this all about?
+
+This repository hosts the code and docs
+needed to integrate with the products offered by
+the Firefox [Application Services](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home) team.
+
+If you're interested in getting involved
+in the development of those products
+then you're in the right place!
+If that's not why you're here
+then you might enjoy:
+
+* The [Application Services Product Portal](https://mozilla.github.io/application-services/),
+  if you're looking to use those products in your application.
+* The [Application Services Team Home](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home)
+  on Mana, if you're trying to find out more about how we build them.
 
 
 ### Overview
 
-The following diagram describes how some of the components hosted here relate to each other within an app.
+This repository is used 
+to build client-side libraries
+for integrating with Firefox Application services
+such as Firefox Accounts, Firefox Sync and Push.
+Each of these is called a "component"
+and is built using a core of shared code
+writen in Rust, exposed over a C FFI
```

Comment:
Hm, it's probably nitpicking, but C FFI is a weird description for it. We do have a small amount of C code, but just a header file to support Swift's FFI (those are in the 'swift bindings' bits).

Calling it "Rust's FFI" is more accurate. Or a "C-compatible ABI", I guess.

Or I guess C FFI is fine too, it just implies to me that there's actually C code in it, when there isn't (ignoring the header file that swift demands).